### PR TITLE
bump k8s to 1.25.2, openshift dependencies and go to 1.19

### PR DESCRIPTION
**Pull request by atiratree**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,43 +4,43 @@ go 1.18
...
-	k8s.io/apimachinery v0.24.0
-	k8s.io/apiserver v0.24.0
-	k8s.io/client-go v0.24.0
-	k8s.io/component-base v0.24.0
-	k8s.io/controller-manager v0.24.0
-	k8s.io/klog/v2 v2.60.1
-	k8s.io/kubectl v0.24.0
-	k8s.io/kubernetes v1.24.0
+	k8s.io/api v0.25.2
+	k8s.io/apimachinery v0.25.2
```
Comment (ingvagabund) [inline go.mod:13]: 0.25.0?

Comment (soltysh) [reply go.mod:13]: I'd suggest all `v0.25.2` rather, we want the latest one available.

**Review by soltysh — APPROVED** [review]
/approve
/assign @ingvagabund 
he has the final tag

File: go.mod (hunk 1 above)
Comment (atiratree) [reply go.mod:13]: updated to `v0.25.2`

**Pull request merged**